ui/components/multichain/import-token-link/import-token-link.tsx

```diff
@@ -20,7 +20,10 @@ import {
   getIsTokenDetectionInactiveOnMainnet,
 } from '../../../selectors';
 
-export const ImportTokenLink = ({ className, ...props }) => {
+export const ImportTokenLink: React.FC<{ className: string }> = ({
+  className,
+  ...props
```

Comment:
We've found that spreading props on the `Box` causes a type error. Do you see a squiggly here? @garrettbear solved this by typing the spread props as `BoxProps` which you can import from the box folder 
```
import { BoxProps } from '../../component-library/box';`
```

```suggestion
  {...(props as BoxProps<'div'>)}
```

You could also change this to a generic type if you want to be able to use the `as` prop on this component

Reply:
@georgewrmarshall I am not seeing this warning here